### PR TITLE
upgrade to java 11 / wicket 9.5

### PR DESCRIPTION
**Pull request by kbachl**
@martin-g : 
can you please have a first look? I needed to uncomment the OSGI as I couldnt get it to work - is this using someone anyway? maybe remove it?

- Wicket 9.5.0
- removed OSGI as it wouldnt work?

### CODE REVIEW

**Comment by martin-g** [comment]
Please define "removed OSGI as it wouldnt work". What did you try ?

File: brix-core/src/main/java/org/brixcms/web/nodepage/BrixNodePageRequestHandler.java [hunk 1]
```diff
@@ -143,16 +144,14 @@ public PageParameters getPageParameters() {
...
-            return page == null;
-        }
 
         @Override
         public Class<? extends IRequestablePage> getPageClass() {
             return BrixNodeWebPage.class;
         }
 
+
+
```
Comment (martin-g) [inline brix-core/src/main/java/org/brixcms/web/nodepage/BrixNodePageRequestHandler.java:154]: not really needed

File: brix-core/src/main/java/org/brixcms/web/nodepage/BrixNodePageRequestHandler.java [hunk 2]
```diff
@@ -143,16 +144,14 @@ public PageParameters getPageParameters() {
             }
         }
 
-        @Override
-        public boolean isNewPageInstance() {
-            return page == null;
-        }
```
Comment (martin-g) [inline brix-core/src/main/java/org/brixcms/web/nodepage/BrixNodePageRequestHandler.java:149]: Is this related to Wicket 9.x ?

File: pom.xml [hunk 3]
```diff
@@ -2,11 +2,11 @@
...
-	<parent>
-		<groupId>org.sonatype.oss</groupId>
-		<artifactId>oss-parent</artifactId>
-		<version>7</version>
-	</parent>
+<!--	<parent>-->
+<!--		<groupId>org.sonatype.oss</groupId>-->
+<!--		<artifactId>oss-parent</artifactId>-->
+<!--		<version>7</version>-->
+<!--	</parent>-->
```
Comment (martin-g) [inline pom.xml:9]: Why ?

File: pom.xml [hunk 4]
```diff
@@ -484,7 +484,7 @@
 				<plugin>
 					<groupId>org.apache.felix</groupId>
 					<artifactId>maven-bundle-plugin</artifactId>
-					<version>2.5.3</version>
+					<version>4.2.1</version>
```
Comment (martin-g) [inline pom.xml:487]: If all the rest are commented out then this one might be commented out as well.

File: brix-core/src/main/java/org/brixcms/web/nodepage/BrixNodePageRequestHandler.java (hunk 2 above)
Comment (kbachl) [reply brix-core/src/main/java/org/brixcms/web/nodepage/BrixNodePageRequestHandler.java:149]: yep - this function isn't existing anymore in wicket 9, so we cant override it

File: pom.xml (hunk 3 above)
Comment (kbachl) [reply pom.xml:9]: my bad, should be version 9

**Comment by kbachl** [comment]
regarding OSGI: I somehow had problems with it and jigsaw at the start, however not sure if this was due to too old bundle plugin version - However: is anyone using this at all?

**Comment by martin-g** [comment]
> However: is anyone using this at all?

I doubt it!
Maybe @briantopping has some concerns about removing it.
https://github.com/brix-cms/brix-cms/commit/f9925071482b8cce9cc12d59a06173c2460f1b3b

**Comment by kbachl** [comment]
@martin-g : not needed anymore, fixed it for osgi now, was my bad as I mixed something up - problem was mainly jigsaw and versions, also jackrabbit got a jdk 8++ cure (it did work for tests but wasnt ok according to the jackrabbit folks); think this works now
-> how do we wanna tag the current java 8 version? - can u branch current master to wicket-8? 
For brix 9 I'd really like a release then :)

**Comment by martin-g** [comment]
>  can u branch current master to wicket-8?

No. I am not a member of brix-cms org/repo anymore.

**Comment by kbachl** [comment]
@martin-g oh,ok
Well... then I have to merge this and make a branch from earlier commit myself later;

**Comment by briantopping** [comment]
Not I. Was a good run.

**Comment by briantopping** [comment]
No concerns, not using it

**Pull request merged**